config/plugins/visualizations/charts/config/charts.xml

```diff
@@ -5,6 +5,7 @@
         <data_source>
             <model_class>HistoryDatasetAssociation</model_class>
             <test type="isinstance" test_attr="datatype" result_type="datatype">tabular.Tabular</test>
+            <test type="isinstance" test_attr="datatype" result_type="datatype">tabular.CSV</test>
```

Comment:
I have a question here - why is it not possible to specify superclass that isn't in the the file extension registry?
Is it designed like this for a reason or just a shortcoming?